source/_integrations/nest.markdown

```diff
@@ -45,36 +45,94 @@ You are in control of the information and capabilities exposed to Home Assistant
 Others devices like Smoke and CO Alarms or Security systems are not currently
 supported by the SDM API.
 
-## Account Setup
+The full detailed instructions for account setup are available in the [Device Access Registration](https://developers.google.com/nest/device-access/registration) Quick Start Guide. The instructions below are included to make this complex setup process a bit easier to follow.
 
-You will need to follow the instructions in [Device Access Registration](https://developers.google.com/nest/device-access/registration) to authorize access to your devices. Follow these steps in the Quick Start Guide:
+## Device Access Regsitration
 
-- Accept the Terms of Service.
-- Pay a fee (currently US$5).
-- [Register for Device Access](https://developers.google.com/nest/device-access/get-started#register_for_device_access) to get a `project_id`.
-- [Set up Google Cloud Platform](https://developers.google.com/nest/device-access/get-started#set_up_google_cloud_platform) and enable the
-  API
-- Authorize your Google Account and create OAuth credentials to get a `client_id` and `client_secret`.
-- Configure *Authorized redirect URIs* with the URL you use to access Home Assistant including the path for the OAuth callback , e.g. `https://<your_home_assistant_url_or_local_ip>:<port>/auth/external/callback` (also see `redirect_url_mismatch` troubleshooting below)
+For the first phase, you will turn on the API and create necessary credentials to have Home Assistant talk to the Nest API.
 
-<div class='note warning'>
-It is currently not possible to share/be invited to a home with a G-Suite account. Make sure that you pay the fee with an account that has access to your devices.
-</div>
+1. First go to the [Device Access Registration](https://developers.google.com/nest/device-access/registration) page.  Click on the button **Go to the Device Access Console**.
+
+    <img src='/images/integrations/nest/device_access.png'>
```

Comment:
Any reason why you are not using markdown for this?

```
![Description of the thing](/images/bla.png)
```

Also, if you mix HTML and Markdown, you need to leave at least one empty new line in between. This allows the parser to switch context.

Reply:
Switched to markdown (was just using html as a copy of what another integration was doing)